tests/integration/conftest.py

```diff
@@ -407,26 +430,31 @@ def missing_os_release_package_workaround(shell):
     yield
 
     os_to_pkg_mapping = {
-        "oracle-7": "oracle*-release-el7",
-        "oracle-8": "oraclelinux-release-el8",
-        "centos-7": "centos-release",
-        "centos-8": "centos-linux-release",
-        "alma-8": "almalinux-release",
-        "rocky-8": "rocky-release",
+        "oracle-7": ("oraclelinux-release-el7", "oraclelinux-release"),
+        "oracle-8": ("oraclelinux-release-el8", "oraclelinux-release"),
+        "centos-7": ("centos-release",),
+        "centos-8": ("centos-linux-release",),
+        "alma-8": ("almalinux-release",),
+        "rocky-8": ("rocky-release",),
```

Comment:
Don't want to be picky, but this could be a list instead... It does not hurt to be a tuple, but a list would have the same effect, and maybe even straightforward to look at and see that we are "iterating over the results"

Reply:
Makes sense, I'm not sure why I went for tuple to be honest :man_shrugging: